kloppy/infra/serializers/event/sportec/deserializer.py

```diff
@@ -74,6 +66,106 @@ def _team_from_xml_elm(team_elm) -> Team:
     return team
 
 
+SPORTEC_FPS = 25
+
+
+class SportecMetadata(NamedTuple):
+    score: Score
+    teams: List[Team]
+    periods: List[Period]
+    x_max: float
+    y_max: float
+    fps: int
+
+
+def sportec_metadata_from_xml_elm(match_root) -> SportecMetadata:
+    """
+    Load metadata from Sportec XML element. This part is shared between event- and tracking data.
+    In the future this might move to a common.sportec package that provides functionality for both
+    deserializers.
+    """
+    x_max = float(match_root.MatchInformation.Environment.attrib["PitchX"])
+    y_max = float(match_root.MatchInformation.Environment.attrib["PitchY"])
+
+    team_path = objectify.ObjectPath("PutDataRequest.MatchInformation.Teams")
+    team_elms = list(team_path.find(match_root).iterchildren("Team"))
+
+    home_team = away_team = None
+    for team_elm in team_elms:
+        if team_elm.attrib["Role"] == "home":
+            home_team = _team_from_xml_elm(team_elm)
+        elif team_elm.attrib["Role"] == "guest":
+            away_team = _team_from_xml_elm(team_elm)
+        else:
+            raise DeserializationError(
+                f"Unknown side: {team_elm.attrib['Role']}"
+            )
+
+    if not home_team:
+        raise DeserializationError("Home team is missing from metadata")
+    if not away_team:
+        raise DeserializationError("Away team is missing from metadata")
+
+    (home_score, away_score,) = match_root.MatchInformation.General.attrib[
+        "Result"
+    ].split(":")
+    score = Score(home=int(home_score), away=int(away_score))
+    teams = [home_team, away_team]
+
+    if len(home_team.players) == 0 or len(away_team.players) == 0:
+        raise DeserializationError("LineUp incomplete")
+
+    # The periods can be rebuild from event data. Therefore, the periods attribute
+    # from the metadata can be ignored. It is required for tracking data.
+    other_game_information = (
+        match_root.MatchInformation.OtherGameInformation.attrib
+    )
+    periods = [
+        Period(
+            id=1,
+            start_timestamp=10_000 / SPORTEC_FPS,
```

Comment:
It is not immediately clear to me where the `10_000`, `100_000`, `200_000` and `250_000` constant values come from. I suspect it is a Sportec convention. How about moving these constant values to a variable with a more descriptive name? Or adding a quick comment explaining the convention?